src/resources/views/crud/filters/date_range.blade.php

```diff
@@ -12,20 +25,8 @@ class="nav-item dropdown {{ Request::get($filter->name)?'active':'' }}">
 		        </div>
 		        <input class="form-control pull-right"
 		        		id="daterangepicker-{{ str_slug($filter->name) }}"
-		        		type="text"
-		        		@if ($filter->currentValue)
-							@php
-								$dates = (array)json_decode($filter->currentValue);
-								$start_date = $dates['from'];
-								$end_date = $dates['to'];
-					        	$date_range = implode(' ~ ', $dates);
-					        	$date_range = str_replace('-', '/', $date_range);
-					        	$date_range = str_replace('~', '-', $date_range);
-
-					        @endphp
-					        placeholder="{{ $date_range }}"
```

Comment:
I see you've removed the part where we manually specify the placeholder. Are you sure this isn't needed? Would the JS plugin do this itself?

Reply:
Indeed the plugin would load the correct formated date.